src/features/nft/addNFT/afterUploadStep/AfterUploadStep.tsx

```diff
@@ -94,20 +94,22 @@ export default function UploadStep({
       rightColumnClass='w-[355px]'
       rightColumnContent={
         <div className='h-full flex justify-between flex-col'>
-          <div className='text-gray-71 font-medium pt-1'>
-            File downloaded successfully, you can continue to configure your
-            incredible NFT
+          <div className='text-gray-71 font-medium'>
+            <p className='text-lg text-gray-4a'>File uploaded sucessfully!</p>
```

Comment:
Typo in word `sucessfully`, I highly suggest to use some editor extension to check for typos/grammar.

Reply:
fixed